heat/core/dndarray.py

```diff
@@ -1319,6 +1319,10 @@ def flatten(self):
         flattened : ht.DNDarray
             The flattened tensor
 
+        See Also
+        ---------
+        flatten
```

Comment:
can you link the flatten docs in manipulations?

:function:`~heat.core.manipulations.flatten`